test/e2e/storage/external/README.md

```diff
@@ -0,0 +1,49 @@
+When a test suite like Kubernete's test/e2e/e2e.test includes this
```

Comment:
Kubernetes's?

Reply:
I think it's "Kubernetes' test/e2e/e2e.test". https://data.grammarbook.com/blog/apostrophes/apostrophes-with-words-ending-in-s/

But let's better avoid that altogether with "like test/e2e/e2e.test from Kubernetes"... done.